OpenRA.Game/World.cs

```diff
@@ -167,8 +167,9 @@ public IOrderGenerator OrderGenerator
 		}
 
 		public readonly ISelection Selection;
+		private readonly IDefaultOrderGenerator defaultOrderGenerator;
```

Comment:
Style nit: No `private` keyword since that is the default visibility.

Reply:
Done